scootapi/server/setup.go

```diff
@@ -92,3 +95,21 @@ func RunServer(bag *ice.MagicBag, schema jsonconfig.Schema, config []byte) {
 	}()
 	log.Fatal("Error serving: ", <-errCh)
 }
+
+// GetConfigText finds the right text for a configFlag.
+// If configFlag looks like a filename (of the form foo.bar where foo and bar are just alphanumeric),
+// read it as an asset.
+// Otherwise, assume it's the literal json text.
+func GetConfigText(configFlag string, asset func(string) ([]byte, error)) ([]byte, error) {
+	if matched, _ := regexp.Match(`[[:alnum:]]*\.[[:alnum:]]`, []byte(configFlag)); matched {
```

Comment:
Assuming configFlag is a whole flattened json string, this will match '{"key": "text.text"}', which is valid.  This actually matches more subtle stuff like:
'{["key": "value"], ["key": ".value"]}', which is kind of what I am getting at